content/authors/ehis-edemakhiota/index.md

```diff
@@ -2,4 +2,6 @@
 title: Ehis Edemakhiota
 type: authors
 ---
-Ehis is a Software Engineer. He is passionate about problem-solving. He enjoys listening to music, watching movies, and running in his leisure time.
+Ehis is a Software Engineering student at Semicolon Africa.
+He is passionate about problem-solving.
+He enjoys listening to music, watching movies,running and shopping online.
```

Comment:
You have edited your author bio, but you placed sentences in different lines. Make it a paragraph.

Reply:
Okay.
Thank you

Reply:
@ehizman, you didn't fix this one. Put all those sentences in one paragraph